src/main/java/thePackmaster/cards/pickthemallpack/AbstractPickThemAllCard.java

```diff
@@ -0,0 +1,51 @@
+package thePackmaster.cards.pickthemallpack;
+
+import thePackmaster.cards.AbstractPackmasterCard;
+import thePackmaster.util.creativitypack.onGenerateCardMidcombatInterface;
+
+public abstract class AbstractPickThemAllCard extends AbstractPackmasterCard implements onGenerateCardMidcombatInterface
+{
+    public boolean combatVersion;
+
+    public AbstractPickThemAllCard(String cardID, int cost, CardType type, CardRarity rarity, CardTarget target)
+    {
+        super(cardID, cost, type, rarity, target, "utility");
+        this.combatVersion = false;
+        this.rawDescription = cardStrings.DESCRIPTION + this.getPickupDescription();
+        this.initializeDescription();
+    }
+
+    @Override
+    public void upp() {
+    }
+
+    @Override
+    public final void uDesc() {
```

Comment:
The purpose of all the code here (including in the constructor) is to allow separation between the base description and the pickup effect description, so that cards in this pack can automatically remove the pickup effect description during combat.